Tests/PackageLoading/ConventionTests.swift

```diff
@@ -440,6 +440,95 @@ class ConventionTests: XCTestCase {
         }
     }
 
+    func testValidSources() throws {
+        var fs = InMemoryFileSystem()
+        try fs.createEmptyFiles("/main.swift",
+                                "/noExtension",
+                                "/Package.swift",
+                                "/.git/anchor",
+                                "/.xcodeproj/anchor",
```

Comment:
Should this be a `.swift` to verify we ignore such files in ignored directories?